content/en/docs/languages/dotnet/metrics/exemplars.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Using exemplars
+linkTitle: Exemplars
+description:
+  Learn how to use exemplars to link metrics with traces in OpenTelemetry .NET
+weight: 40
+---
+
+[Exemplars](/docs/specs/otel/metrics/sdk/#exemplar) are example data points for
+aggregated data. They provide specific context to otherwise general
+aggregations. One common use case is to gain the ability to correlate metrics to
+traces (and logs).
+
+This guide demonstrates how to use exemplars with OpenTelemetry .NET to connect
+metrics and traces, using Prometheus, Jaeger, and Grafana.
+
+## What are exemplars?
+
+Exemplars represent individual measurements that are part of an aggregated
+metric. They allow you to:
+
+- Link metrics to traces that were active when the measurement was taken
+- Identify outliers or interesting data points within aggregated metrics
+- Better understand the causes of metric changes by exploring associated traces
+
+## Components used in this guide
+
+- **OpenTelemetry .NET SDK**: Instrumentation for your application
+- **Prometheus**: Metrics backend that supports exemplars
+- **Jaeger**: Distributed tracing backend
+- **Grafana**: UI to query metrics and traces, and navigate between them using
+  exemplars
+
+## Setup
+
+### Install and run Jaeger
+
+1. Download the
+   [latest binary distribution](https://www.jaegertracing.io/download/) of
+   Jaeger
+2. Extract it to a local directory
+3. Run the `jaeger-all-in-one(.exe)` executable:
+
+```shell
+./jaeger-all-in-one --collector.otlp.enabled
+```
+
+### Install and run Prometheus
+
+1. Download the [latest release](https://prometheus.io/download/) of Prometheus
+2. Extract it to a local directory
+3. Run Prometheus with the required feature flags:
+
+```shell
+./prometheus --enable-feature=exemplar-storage --web.enable-otlp-receiver
+```
+
+### Install and configure Grafana
+
+1. Follow the
+   [operating system specific instructions](https://grafana.com/docs/grafana/latest/setup-grafana/installation/#supported-operating-systems)
+   to install Grafana
+2. Start the Grafana server
+3. Open [http://localhost:3000/](http://localhost:3000/) in your browser
+4. Log in with the default credentials (admin/admin)
+5. Configure data sources:
```

Comment:
We also have a self-contained Docker image for testing/demo purposes that includes OTel-compatible ingestion endpoints: https://github.com/grafana/docker-otel-lgtm